### ISSUE TITLE
Accessing admin console gives 302

### ISSUE THREAD

**Issue by vidursatija** [antoniomika/sish#137]
Hi,

I enabled the admin console and changed the auth token. When I try to access `http(s)://domain/_sish/console?x-authorization=<>` it gives me a 302 and redirects me here. Am I doing anything wrong? 

Thanks!

**Comment by antoniomika** [comment]
Hmm, this is generally due to an auth code issue (i.e. it's not valid)

**Comment by vidursatija** [comment]
I crossed check the validity and it wasn't a problem. Following the console output:
```shell
2020/10/09 - 06:27:55 | Error retrieving keepalive response: EOF
2020/10/09 - 06:27:55 | Closed SSH connection for: 180.151.147.247:51260 user: vidursatija
2020/10/09 - 06:27:58 | http: TLS handshake error from 180.151.147.247:51288: cannot find connection for host: <domain>
2020/10/09 - 06:27:59 | http: TLS handshake error from 180.151.147.247:51290: cannot find connection for host: <domain>
2020/10/09 - 06:27:59 | http: TLS handshake error from 180.151.147.247:51292: tls: client offered only unsupported versions: [301]
2020/10/09 - 06:27:59 | http: TLS handshake error from 180.151.147.247:51294: cannot find connection for host: <domain>
2020/10/09 - 06:27:59 | http: TLS handshake error from 180.151.147.247:51296: cannot find connection for host: <domain>
2020/10/09 - 06:28:00 | http: TLS handshake error from 180.151.147.247:51298: tls: client offered only unsupported versions: [301]
```
